shared/src/packages/articles/libs/types/article-with-author.type.ts

```diff
@@ -11,6 +11,7 @@ type ArticleWithAuthorType = ArticleEntityType & {
     prop: string | null;
   } | null;
   genre?: string | null;
+  coverId: number | null;
```

Comment:
`coverId` is present in ArticleEntityType

Reply:
Yes it is, but I use this type for the constructor so I can't omit it

Reply:
Don't understand. You can just remove it and nothing will change in this type

Reply:
Oh, I forgot that this type is extended by ArticleEntityType, thank you